packages/data-point/lib/entity-types/integrations.test.js

```diff
@@ -222,34 +223,39 @@ test('Model Entity Instance', () => {
 })
 
 describe('trace feature', () => {
-  test('trace via options parameter', () => {
-    const consoleTime = console.time
-    const consoleTimeEnd = console.timeEnd
+  let mockDateNow
+  let mockWriteFileP
+  afterEach(() => {
+    mockDateNow.mockRestore()
+    mockWriteFileP.mockRestore()
+  })
 
-    const timeIds = []
-    console.time = id => {
-      timeIds.push({
-        type: 'time',
-        id: id
-      })
-    }
-    console.timeEnd = id => {
-      timeIds.push({
-        type: 'timeEnd',
-        id: id
+  test('trace via options parameter', () => {
+    let calls = 0
+    const NS_PER_SEC = 1e9
+    jest.spyOn(process, 'hrtime').mockImplementation(t => {
```

Comment:
is this mock getting removed after it's used?

Reply:
good call, thanks,  removing it